sched/adaptdl_sched/config.py

```diff
@@ -57,3 +57,13 @@ def get_storage_subpath():
 def get_job_default_resources():
     val = os.getenv("ADAPTDL_JOB_DEFAULT_RESOURCES")
     return json.loads(val) if val is not None else None
+
+
+def get_job_pod_patch():
+    val = os.getenv("ADAPTDL_JOB_POD_PATCH")
+    return json.loads(val) if val is not None else None
+
+
+def get_job_container_patch():
```

Comment:
Keep getter names matching env var names: `get_job_patch_pods()` and `get_job_patch_containers()`